### PR TITLE
Added Atocs and associated items.

### PR DESCRIPTION
**Pull request by Nightspark43**
Added Atocs and a few related items, plus fixed a typo I made a while back, you said we don't need Changelogs for updates before launch so...

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -202,7 +202,7 @@
 	desc = "A box containing 5 highly experimental supermatter grenades."
 	icon_state = "radbox"
 	startswith = list(/obj/item/weapon/grenade/supermatter = 5)
-    
+
```
Comment (Nightspark43) [inline code/game/objects/items/weapons/storage/boxes.dm:205]: No idea what this here's about, I checked the .DM, nothing moved above the new Atocboxes.

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -281,6 +281,11 @@
 	desc = "Drymate brand neaera cubes, shipped from Jargon 4. Just add water!"
 	startswith = list(/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped/neaeracube = 5)
 
+/obj/item/weapon/storage/box/monkeycubes/atoccubes
+	name = "atoc cube box"
+	desc = "Drymate brand neaera cubes, shipped from Avis. Just add water!"
```
Comment (Nightspark43) [inline code/game/objects/items/weapons/storage/boxes.dm]: And just noticed this typo, whoops.

**Review by BlueNexus — APPROVED** [review]

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 2 above)
Comment (BlueNexus) [inline code/game/objects/items/weapons/storage/boxes.dm]: Change this to Atoc, not neaera.

**Review by BlueNexus — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 2 above)
Comment (Nightspark43) [reply code/game/objects/items/weapons/storage/boxes.dm]: Just did.

**Review by BlueNexus — APPROVED** [review]

**Pull request merged**